projects/storefrontapp-e2e-cypress/cypress/integration/regression/billing-address.e2e-spec.ts

```diff
@@ -37,21 +37,20 @@ context('Payment billing address', () => {
   it('should go to checkout', () => {
     checkout.registerUser();
     checkout.signOutUser();
-    checkout.goToProductDetailsPage();
-    checkout.addProductToCart();
-    checkout.loginUser();
+    checkout.goToCheapProductDetailsPage();
+    checkout.addCheapProductToCartAndLogin();
```

Comment:
`checkout.loginUser()` was removed.  Don't we still need to sign in the user here?

Reply:
The new function `addCheapProductToCartAndLogin()` contains the login function.
This was done so that we can add a `cy.wait()` for the shipping page after the login happens.